packages/cli/openapi-parser/src/v3/extensions/extensions.ts

```diff
@@ -1,41 +1,75 @@
-/**
- * Used to detect if an endpoint has an async version of it
- * Example usage:
- *   paths:
- *    /path/to/my/endpoint:
- *      x-fern-async-config:
- *        discriminant:
- *          type: header
- *          name: X-Header-Name
- *          value: async
- *        response-status-code: 202
- **/
-export const X_FERN_ASYNC_CONFIG = "x-fern-async-config";
+import { Values } from "@fern-api/core-utils";
 
-/**
- * Used to create veriables in the fern definition
- * Example usage:
- * x-fern-sdk-variables:
- *   appName:
- *     type: string
- * paths:
- *   /path/to/my/endpoint/{id}:
- *     parameters:
- *       - name: id
- *         in: path
- *         type: string
- *         x-fern-sdk-variable: appName
- */
-export const X_FERN_SDK_VARIABLES = "x-fern-sdk-variables";
-export const X_FERN_SDK_VARIABLE = "x-fern-sdk-variable";
+export const FernOpenAPIExtension = {
+    SDK_METHOD_NAME: "x-fern-sdk-method-name",
+    SDK_GROUP_NAME: "x-fern-sdk-group-name",
+    STREAMING: "x-fern-streaming",
```

Comment:
this is a boolean 

```
paths: 
  /my-path
    post: 
      x-fern-streaming: true
```
This will set `response-stream` in fern with the success response type (200 status code)